source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -285,6 +286,14 @@
                             text: itemNumber + " - " + item.Text,
                             description: item.Description,
                             isRowSelected: rowSelected,
+                            addSelfComment: function (data, event) {
+                                _.find(this.comments, function (selfComment) {
```

Comment:
I am not sure to understand, but it seems like you are using `_.find()` as a _foreach_.

The function parameter on _find_ is supposed to be a predicate to detect the item that you are finding, see http://underscorejs.org/#find